### DISCUSSION TITLE
On a PicoCNC V1.52 board, where can I tap +5V power?

### DISCUSSION THREAD

**Discussion by TPMoyer** [grblHAL/RP2040#95] [General]
Have a Version 1.52 of the PicoCNC board, and unlike the V1.53, the V1.52 does not have the +5V aux power pins.    Is there somewhere on a V1.52 board that can be tapped for +5V aux power.    Am trying to do a 1 for 1 swap: new PicoCNC for an old arduino uno.   The +5V load would be the 3X DRV8825 Stepper Motor Driver Modules on a Protoneer type CNC hat

**Comment by terjeio** [comment]
I do not have a v1.52 board at hand, the v1.52eth I have has 5V exposed at the Relay Voltage header - and if the jumper is set to 5V I believe it is also available at the +V Relays connectors.
You may also check the schematics or ask @phil-barrett  at the [board homepage](https://github.com/phil-barrett/PicoCNC).

**Reply by phil-barrett** [reply]
Yes, that is correct. That's why I added the aux 5v header.

**Comment by TPMoyer** [comment]
Thank You.     Appreciate the quick and complete response.
